integrated_channels/integrated_channel/management/commands/__init__.py

```diff
@@ -45,6 +45,7 @@
     for integrated_channel_class in (
         BlackboardEnterpriseCustomerConfiguration,
         CanvasEnterpriseCustomerConfiguration,
+        CornerstoneEnterpriseCustomerConfiguration,
```

Comment:
Cornerstone would now be included in the regularly scheduled transmit content task

Reply:
you need to update the comment on line 41/42